docs/endpoints.rst

```diff
@@ -132,34 +147,40 @@ To list available endpoints on the current system, run::
 
 Endpoints can be the following states:
 
-* **Initialized**: This status means that the endpoint has been created, but not started
+* **Initialized**: the endpoint has been created, but not started
   following configuration and not registered with the `funcx service`
-* **Running**: This status means that the endpoint is active and available for executing
-  functions
-* **Stopped**: This status means that the endpoint was stopped by the user. It is not running
-  right now and therefore, cannot service any functions. It can be started again without issues.
-* **Disconnected**: This status means that the endpoint disconnected unexpectedly. It is not running
-  right now and therefore, cannot service any functions. Starting this endpoint will first invoke
+* **Running**: the endpoint is active and available for executing  functions
+* **Stopped**: the endpoint was stopped by the user. It is not running
+  and therefore, cannot service any functions. It can be started again without issues.
+* **Disconnected**: the endpoint disconnected unexpectedly. It is not running
+  and therefore, cannot service any functions. Starting this endpoint will first invoke
   necessary endpoint cleanup, since it was not stopped correctly previously.
 
 
 Container behaviors and routing
 -------------------------------
 
-The funcX endpoint supports various technologies of containers (e.g., docker and singularity) and also raw processes on clusters (e.g., laptop, Kubernetes, and Supercomputers).
-The funcX endpoint also supports different routing mechanisms for different use cases.
+The funcX endpoint can run functions using indepenent Python processes or optionally
+inside containers. funcX supports various container technologies (e.g., docker and singularity)
+and different routing mechanisms for different use cases.
 
 Raw worker processes (`worker_mode=no_container`):
 
-* Hard routing: All worker processes are of the same type "RAW". It this case, the funcx endpoint simply routes function tasks to any available worker processes. The is the default mode of a funcx endpoint.
+* Hard routing: All worker processes are of the same type "RAW". It this case, the funcx endpoint simply routes tasks to any available worker processes. This is the default mode of a funcx endpoint.
 * Soft routing: It is the same as hard routing.
```

Comment:
```suggestion
* Soft routing: For raw worker processes, this is the same as hard routing.
```

Reply:
would it make more sense to say soft routing is not supported in this case?  I don't think I understand how soft and hard can be the same

Reply:
or maybe hard and soft aren't really very good terms for the distinction we are trying to make here

Reply:
We struggled a bit with naming here. I think we should come back to this in a separate pass to avoid a hold up.